assets/stylesheets/main.scss

```diff
@@ -18,14 +15,14 @@ $cg-blue: #1f2c38;
 }
 
 .logo {
-  width: auto;
   float: left;
-}
+  width: auto;
 
-.logo h1 a {
-  text-decoration: none;
-  font-size: 2rem;
-  color: #595959;
+  h1 a {
```

Comment:
Does this need the `h1` or does just `a` work?

Is this something that should be added to cg-style? I think the rule kinda makes sense there, I can add.